Assignments/A3/Distro/QLearn.h

```diff
@@ -21,21 +21,21 @@
 
 // Generally needed includes
 // #include <malloc.h>
+#include <assert.h>
 #include <math.h>
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <assert.h>
 #include <time.h>
 
 #define alpha .01 // Learning rate for standard Q-Learning
 #define lambda .5 // Discount rate for future rewards
 #define max_graph_size 32 * 32
 
-#define numFeatures 5 // UPDATE THIS to be the number of features you have
+#define numFeatures 3 // UPDATE THIS to be the number of features you have
 #define numActions 4
-#define BIG_DBL 1000000.0
+#define BIG_DBL 1000000000000.0
```

Comment:
you can change this back; I increased it when I encountered the ever increasing weights issue